src/libraries/System.IO.Pipes/ref/System.IO.Pipes.csproj

```diff
@@ -1,13 +1,25 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)</TargetFrameworks>
+    <!-- The non NetCoreAppCurrent tfms never ship, they are just used to produce reference assemblies,
+         to expose types for Pipes.AccessControl to forward to. -->
+    <TargetFrameworks>$(NetCoreAppCurrent);net5.0;netcoreapp3.1</TargetFrameworks>
     <Nullable>enable</Nullable>
   </PropertyGroup>
+  <!-- DesignTimeBuild requires all the TargetFramework Derived Properties to not be present in the first property group. -->
+  <PropertyGroup>
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'netcoreapp3.1'">4.1.2.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetFramework)' == 'net5.0'">5.0.0.0</AssemblyVersion>
+  </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.IO.Pipes.cs" />
+    <Compile Include="System.IO.Pipes.Internal.cs" Condition="'$(TargetFramework)' != '$(NetCoreAppCurrent)'" />
```

Comment:
I guess this name is OK.  I couldn't think of anything better.

Reply:
I scanned existing patterns of that and found couple of ".Internal" files so I renamed it from ".Internals" to ".Internal".